apiDefinition.swagger.json

```diff
@@ -0,0 +1,245 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Impower",
+    "description": "Connector to Upload Invoices to Impower",
+    "version": "1.0",
+    "contact": {
+      "name": "Impower GmbH",
+      "url": "https://www.impower.de",
+      "email": "info@impower.de"
+    }
+  },
+  "host": "api.app.impower.de",
+  "basePath": "/services",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [],
+  "produces": [],
+  "paths": {
+    "/pmp-accounting/api/v1/invoices/upload": {
+      "post": {
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/InvoiceEpo"
+            }
+          }
+        },
+        "summary": "Rechnung hochladen",
+        "description": "Rechnung an Impower senden",
+        "operationId": "Upload_Invoice",

```

Comment:
@jforscht all operationIds should be in Camel case with no spaces or punctuation. Please see [here](https://learn.microsoft.com/en-us/connectors/custom-connectors/connector-string-guidance#operations).
Please fix all operaionIds in the swagger file.